sdk/core/core/src/az_http_policy_retry.c

```diff
@@ -141,30 +141,30 @@ AZ_INLINE AZ_NODISCARD az_result _az_http_policy_retry_get_retry_after(
 }
 
 AZ_NODISCARD az_result az_http_pipeline_policy_retry(
-    _az_http_policy* policies,
-    void* options,
```

Comment:
And I think Jeffrey wanted to name it options rather than data, check with him.

Reply:
Yes, policy's have 'options'; not 'data'